models/packages/package_version.go

```diff
@@ -298,8 +299,7 @@ func SearchVersions(ctx context.Context, opts *PackageSearchOptions) ([]*Package
 	return pvs, count, err
 }
 
-// SearchLatestVersions gets the latest version of every package matching the search options
-func SearchLatestVersions(ctx context.Context, opts *PackageSearchOptions) ([]*PackageVersion, int64, error) {
+func getLatestVersions(ctx context.Context, opts *PackageSearchOptions) *xorm.Session {
```

Comment:
I'm not really pleased by this name. Maybe `buildLatestVersionsSession` or something similar?